resources/ext/config/conf.d/puppetserver.conf

```diff
@@ -2,7 +2,7 @@
 jruby-puppet: {
     # Where the puppet-agent dependency places puppet, facter, etc...
     # Puppet server expects to load Puppet from this location
-    ruby-load-path: [/opt/puppetlabs/puppet/lib/ruby/vendor_ruby]
+    ruby-load-path: [/opt/puppetlabs/puppet/lib/ruby/vendor_ruby, /opt/puppetlabs/puppet/cache/lib]
```

Comment:
Not to create too many threads along the same conversation, and assuming Josh is thinking along the lines of constructing an absolute path to the agent's libdir...

Wouldn't this mean the _only_ way hiera could load backends would be if they're located in the agent plugin sync directory?  Do we have a sense of how end users distribute backend extensions, e.g. are backends located in Ruby gems supported?  If so, an absolute path solution might break support for that behavior, or similar behaviors.